src/celestia/qt/qtcelestialbrowser.cpp

```diff
@@ -188,7 +188,7 @@ QVariant StarTableModel::data(const QModelIndex& index, int role) const
         {
         case NameColumn:
             {
-                uint32_t hipCatNo = star->getMainIndexNumber();
+                uint32_t hipCatNo = star->getIndex();
```

Comment:
*Just a marker for a comment regarding crossindexes here*

Reply:
I dont understand...